examples/hybrid_canister/src/canister.ts

```diff
@@ -0,0 +1,25 @@
+import { Canister, query, serverCanisterMethods, text, update } from 'azle';
+import express from 'express';
+
+export default Canister({
+    ...serverCanisterMethods(serverCallback),
+    candidQuery: query([], text, () => {
```

Comment:
For both purposes of testing and illustrating how to do it, I think it might be good to show either the init or post upgrade method being overwritten from the canister context

Reply:
And I would like to see that as an additional canister, ie (canister_init_and_post_upgrade.ts)

Reply:
I have implemented this additional canister: canister_init_and_post_upgrade.ts

I don't think we need to do either init or post_upgrade for now